genie-core/src/main/java/com/netflix/genie/core/util/MetricsUtils.java

```diff
@@ -80,4 +83,44 @@ private MetricsUtils() {
     public static Map<String, String> newFailureTagsMapForException(final Throwable t) {
         return addFailureTagsWithException(Maps.newHashMap(), t);
     }
+
+    /**
+     * Convenience method to add common tags for slicing and dicing atlas metrics from Job
+     * execution environment to the map of tags passed in.
+     * Populates defaultIdValue in case ids are not present in the job execution environment
+     * E.g defaultIdValue will be added to the tags maps for the cluster_id key in case cluster_id
+     * does not exist in jobExecEnv
+     *
+     * @param tags The map of tags
+     * @param jobExecEnv Genie job execution env
+     * @param defaultIdValue Default value for missing id in jobExecEnv
+     */
+    public static void addCommonMetricTags(final JobExecutionEnvironment jobExecEnv,
```

Comment:
For multiple arguments here that don't fit on one line within the character limit I like to put each parameter on a new line. It makes it easier (at least for me) to read it so I don't miss parameters buried in a list an I feel it looks a bit nicer

Reply:
`CommonMetric` is a vague name. This helper is adding tags related to one specific thing: the job execution environment. Why not have a name that reflect this?